pyproject.toml

```diff
@@ -87,8 +87,12 @@ indent-width = 4
 indent-style = "space"
 line-ending = "lf"
 
+[tool.ruff.lint]
+select = ["E", "F"]
+preview = false
+
 [tool.ruff.lint.pycodestyle]
-max-line-length = 100
+max-line-length = 130
```

Comment:
Consistent with pycodestyle settings.

Still need some work to implement pycodestyle checks.